UserApi/UserApi/Controllers/UserController.cs

```diff
@@ -227,6 +227,10 @@ public async Task<IActionResult> DeleteUser([FromRoute]string username)
             {
                 return NotFound();
             }
+            catch (ForbiddenRequestToRemoveUserException)
+            {
+                return StatusCode((int)HttpStatusCode.Forbidden);
```

Comment:
Could this be 
```suggestion
                return Forbid();
```

I see some reasons why you might not want to here https://stackoverflow.com/questions/45095853/how-to-return-403-forbidden-response-as-iactionresult-in-asp-net-core

Reply:
this was a suggestion from Anai and Arif, initially I was using Forbid, but in the unit tests we couldnt find a nice way to assert it (when you cast it as a ForbidResult Object it doesnt return anything to assert with). Returning the statuscode allowed us to actually assert status code properly and not just assert not null on the result of a cast